tasks/README.md

```diff
@@ -174,4 +174,7 @@ Name | Summary | Category
 `task184_break_generate_question` | Generate a question based on the given steps used to answer it. | Question Generation
 `task170_hotpotqa_answer_generation.json` | Given a set of context and supporting facts, answer the question asked based on them. | Answer Generation
 `task191_hotpotqa_question_generation.json` | Given a set of context, supporting facts and an answer, generate the question asked based on them. | Question Generation
-`task192_hotpotqa_sentence_generation.json` | Given a context paragraph, question and corresponding answer, generate the supporting facts that helps in answering question. | Sentence Generation
\ No newline at end of file
+`task192_hotpotqa_sentence_generation.json` | Given a context paragraph, question and corresponding answer, generate the supporting facts that helps in answering question. | Sentence Generation
+`task210_logic2text_equation_generation.json` | Given a natural language interpretation, generate a command using logical operations | Equation Generation
+`task211_logic2text_classification.json` | Given a command and corresponding interpretation, classify whether it's right interpretation or not | Classification
+`task212_logic2text_classification.json` | Given a command (in the form of logical operators), divide in seven categories based on their action | Classification
```

Comment:
Not sure "divide in seven categories based on their action" means. Rephrase?

Reply:
I rephrased it. Please check and let me know changes.